SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ExpectedViewDelta.cpp

```diff
@@ -102,7 +111,7 @@ bool ExpectedViewDelta::Compare(const ViewDelta& Other)
 	for (int32 i = 0; i < DeltaKeys.Num(); ++i)
 	{
 		ExpectedEntityDelta& LhsEntityDelta = EntityDeltas[DeltaKeys[i]];
-		EntityDelta RhsEntityDelta = RhsEntityDeltas[i];
+		EntityDelta RhsEntityDelta = Other[i];
```

Comment:
maybe better like this:
EntityDelta& RhsEntityDelta = Other[i];

Reply:
done, thanks for the comment- also made them const